lib/converters/converter-v1-to-v2.js

```diff
@@ -363,6 +363,66 @@ _.extend(Builders.prototype, {
     },
 
     /**
+     * Constructs an array of Items & ItemGroups compatible with the V2 format.
+     *
+     * @param collectionV1
+     */
+    itemGroups: function (collectionV1) {
+        var self = this,
+            items = [],
+            itemGroupCache = {};
+
+        // Construct an empty ItemGroup for each folder
+        _.each(collectionV1.folders, function (folderV1) {
+            itemGroupCache[folderV1.id] = {
+                /* jshint ignore:start */ // jscs:disable
+                _postman_id: self.options.retainIds ? folderV1.id : undefined,
+                /* jshint ignore:end */ // jscs:enable
+                name: folderV1.name,
+                description: folderV1.description,
+                item: []
+            };
+        });
+
+        // Populate each ItemGroup with subfolders
+        _.each(collectionV1.folders, function (folderV1) {
+            var itemGroup = itemGroupCache[folderV1.id],
+                hasSubfolders = folderV1.folders_order && folderV1.folders_order.length,
+                hasRequests = folderV1.order && folderV1.order.length;
+
+            // Add subfolders
+            hasSubfolders && _.each(folderV1.folders_order, function (subFolderId) {
+                if (!itemGroupCache[subFolderId]) {
+                    // todo: figure out what to do when a collection contains a subfolder ID,
```

Comment:
Which is what happens right now. The note is there to ensure we keep in mind that this behavior might have to change later.